eval $(ssh-agent -s)

```diff
@@ -0,0 +1,50 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
@RichieWilynnton 
Maybe you need to make a new branch from master, `git cherry-pick` the commits you need, and then `git push --force`.